controllers/controllers.js

```diff
@@ -28,7 +37,13 @@ exports.getArticles = (req, res, next) => {
 
   fetchArticles()
     .then((articles) => {
-      res.status(200).send({ articles });
+      const articlePromises = articles.map((article) => {
+        return fetchArticleById(article.article_id).then((article) => article);
+      });
+
+      Promise.all(articlePromises).then((articles) => {
```

Comment:
why are you doing any of this? `fetchArticles` got you some article and you're sending them back, no? Why are you fetching article by ID afterwards?